src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -5,7 +5,7 @@
 import os
 
 NODE_VERSION_DEFAULT = "10.14"
-NETCORE_VERSION_DEFAULT = "2.2"
```

Comment:
We downgraded this? is that not on the supported list for Windows?
As per portal for windows there is a newer LTS version 
<img width="872" alt="image" src="https://user-images.githubusercontent.com/13839372/86975932-447e3680-c12e-11ea-87b5-aa4934a38670.png">